### ISSUE TITLE
improve handling for dlcs

### ISSUE THREAD

**Issue by trembon** [trembon/switch-library-manager#25]
currently when scanning for dlcs, the game, dlc name and dlc id is stored as a string
this should be better handled as an object and let the presentation layer (gui/console) choose how to present it

**Comment by mspykerez** [comment]
Does this affect the MISSING DLC tab?
Because currently the DLC name and ID are on the same column and I think it's better to have them in separate columns.
A new column with the release date of the DLC is also missing there.

Also notice that DLC updates appear under the MISSING UPDATES tab and perhaps would be better to move them to the MISSING DLC tab to avoid mixing with regular game updates?

Note: DLCs can also have updates, thats why there is a 'Type' column in the MISSING UPDATES tab for Base/DLC

**Comment by trembon** [comment]
yes, it would affect that tab, as currently the DLC data is parsed in a weird way which makes it hard to work with the DLC information in the views
so yeah, this change would allow to improve the columns shown in the missing dlc tab
